packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -238,6 +238,14 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         }
     }
 
+    private _getChannelName() {
+        const channel_name =
+            this._store.getState().config.getConfig()?.storeConfig?.links.siteLink ||
```

Comment:
```suggestion
            this._store.getState().config.getStoreConfigOrThrow().links.siteLink
```

Reply:
Thank you for the suggestion it is a way better approach